client/src/components/Form/Form.scss

```diff
@@ -131,6 +131,14 @@
   &:empty {
     display: none;
   }
+
+  .icon {
+    &::before {
+      vertical-align: middle;
+      margin-right: 6px;
```

Comment:
What happens if the icon is used at the end of the description? 🤔

Reply:
No margin is shown between the text and the icon.

Some potential solutions:
* Wrap text in `<p>` tag to be able to correctly apply margin
* Treat description blurb as single element and expect people to add whitespace `&nbsp` between icons
* Add icon classes to the description element

May be out of scope for this piece of work though.

Reply:
I think putting the icon at the start of the description is likely the most common use-case, and we can improve this later, so happy to proceed for now.